src/content/en/updates/2018/06/chrome-68-deps-rems.md

```diff
@@ -61,6 +55,18 @@ Group and support for it is now removed from Chrome.
 [Chromestatus Tracker](https://www.chromestatus.com/feature/5253912718213120) &#124;
 [Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=845288)
 
+## Deprecations
+
+Current deprecations are listed below. {% include "web/updates/_shared/deprecations.html" %}
```

Comment:
The includes say "Chrome Platform Status provides a list of..." which doesn't make sense when preceded by "Current deprecations are listed below.", I'd recommend removing the "Current deprecations are listed below."

Reply:
I guess I'm not conveying what is actually happening here. The list 'list below' is only features that are newly deprecated in this release. Because features stay deprecated for more than one release I've provided a link to the complete list. So it's: "here's the new ones. Here's all of them."

Reply:
Look at the changes as a whole. I've taken a different approach.